tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -538,7 +538,8 @@ EOT
 	AGENT_DEST="${AGENT_DIR}/${AGENT_BIN}"
 
 	if [ -z "${AGENT_SOURCE_BIN}" ] ; then
-		[ "$ARCH" == "ppc64le" ] && ([ "$LIBC" == "gnu" ] || die "LIBC type for ppc64le should be gnu")
+                # LIBC type for ppc64le should be gnu
+		[ "$ARCH" == "ppc64le" ] && ([ "$LIBC" == "gnu" ] || LIBC=gnu)
 		bash ${script_dir}/../../../ci/install_musl.sh
 		# rust agent needs ${arch}-unknown-linux-${LIBC}
 		rustup show | grep linux-${LIBC} > /dev/null || bash ${script_dir}/../../../ci/install_rust.sh
```

Comment:
yes @clnperez, I understand now. I was in the opinion that make at L557 below would internally override `LIBC` after excluding the L542 above. I also missed this line where `LIBC` type is checked for to install rust and if not set above would consider `LIBC` as musl here which doesn't exist for ppc64le.